scripts/cc-cli/src/commands/receive.ts

```diff
@@ -1,5 +1,6 @@
 import { Command, OptionValues } from "commander";
 import { getSeedFromOptions, initKeyringPair } from "../utils/account";
+import { newApi } from "../api";
 
 export function makeReceiveCommand() {
   const cmd = new Command("receive");
```

Comment:
The name of this command the it's description & implementation don't make sense to me.

If it is meant to display the address of the account they why not call it `show-address` ?